R/getMetadataComment.R

```diff
@@ -0,0 +1,23 @@
+#' getMetadataComment
+#'
+#' Helper function extract a metadata comment
+#'
+#' @md
+#' @param x object the metadata should be extracted from
+#' @param name name of the metadata to be extracted (e.g. unit)
+#' @author Jan Philipp Dietrich
+#' @examples
+#' x <- as.magpie(1)
+#' getComment(x) <- c(" description: example description", " unit: kg")
+#' getMetadataComment(x, "unit")
+#' getMetadataComment(x, "description")
+#' @export
+
+getMetadataComment <- function(x, name) {
+  comment <- attr(x, "comment")
+  key <- paste0("^ ", name, ": ")
+  entry <- grep(key, comment)
+  if (length(entry) == 0) return(NULL)
```

Comment:
Did you consider throwing an error or warning in this case? Maybe one that can be disabled with an argument

Reply:
thought about it, but decided not to implement it at this stage. We can add this feature later in case we need it